### ISSUE TITLE
Failed to compile on Ubuntu focal (20.04.2)

### ISSUE THREAD

**Issue by mpenet** [snyball/Hawck#56]
First of all, thanks for the project, I use it on my personal laptop (ubuntu 20.10) and it's flawless. It really deserves more praise, the model, dsl and implementation are spot on.

I am trying to build 0.7.1 (which works on my other laptop) on my work machine, but it fails to compile somehow

**Expected behavior**

I would expect it to build successfully 

**Desktop (please complete the following information):**
- Distro: Ubuntu 20.04.2 LTS
- Version 0.7.1
- Method of installation (put X inside [ ] if any of the following apply):
  - [x] `install.sh`
  - [ ] AUR package
- Keymap: us
- Keyboard (put X inside [ ] if any of the following apply):
  - [x] USB
  - [ ] Bluetooth
  - [x] Laptop built-in
  - [ ] I use a USB steering wheel exclusively for all my data input.

**Additional context**

The ./install.sh output https://gist.github.com/mpenet/968284b7687aa80233a45e78df4ec36d

On a side note, on my machine with ubuntu 20.10 I tried to build master but it fails at another point. I think it fails to install most things under .local/share/hawck/, I just had symlinks to non-existant files, maybe the recent changes on the install process broke something there, I didn't dig deeper I just went for most recent tag and build it successfully on that machine.

**Comment by mpenet** [comment]
actually it might be the same as https://github.com/snyball/Hawck/issues/50

```
uname -a
Linux wintermute 5.4.0-65-generic #73-Ubuntu SMP Mon Jan 18 17:25:17 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

the other laptop is on 5.8.0-41-generic

**Comment by mpenet** [comment]
Yes confirmed: I cherry-picked 2949221e562db6adfb3413fab31451abc580e56d on v0.7.1 and it now compiles, Feel free to close the issue.

**Comment by snyball** [comment]
@mpenet Good to see that you got it running, I'll mark this as resolved by the time I get a 7.2 version released, which will require some testing on Ubuntu to resolve the installation issues you mentioned.
